### PR TITLE
[DOPS-1896]: Deploy Subquery projects for SORA

### PR DESCRIPTION
**Pull request by f33r0**
# Task

[DOPS-1896]: Deploy Subquery projects for SORA

## Changes

1. Update image
2. Update CI

## Author

Signed-off-by: Nikita Zaporozhets <zaporozhets@soramitsu.co.jp>

[DOPS-1896]: https://soramitsu.atlassian.net/browse/DOPS-1896

### CODE REVIEW

**Review by wpi86 — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-RUN        npm i -g --unsafe-perm @subql/node@0.23.1
+RUN        npm i @subql/node@0.31.1 --location=global
 
 FROM       node:16-alpine
 ARG        SORA_ENDPOINT
+ENV        NODE_VERSION=16.14.2
+ENV        YARN_VERSION=1.22.18
 ENV        TZ utc
-RUN        apk add --no-cache tini git
+RUN        apk add --no-cache tini git curl
```
Comment (wpi86) [inline Dockerfile]: Why do you need curl?

**Review by wpi86 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (f33r0) [reply Dockerfile]: for healthcheck

Comment (wpi86) [reply Dockerfile]: you can use something like https://github.com/soramitsu/nbc-deploy/blob/develop/fastcore/docker-compose.yml#L8

Comment (wpi86) [reply Dockerfile]: and why do you need a git?

Comment (f33r0) [reply Dockerfile]: edited

**Review by wpi86 — APPROVED** [review]

**Review by Cre-eD — COMMENTED** [review]
What is this prod branch means?

**Comment by f33r0** [comment]
> What is this prod branch means?

This means that the prod image is being built from here.

**Comment by Cre-eD** [comment]
> > What is this prod branch means?
> 
> This means that the prod image is being built from here.

But people use master for it, why do you have prod branch?

**Review by Cre-eD — APPROVED** [review]

**Comment by f33r0** [comment]
> > > What is this prod branch means?
> > 
> > 
> > This means that the prod image is being built from here.
> 
> But people use master for it, why do you have prod branch?

This branches created by Boris, not me :)

**Pull request closed without merging**